epub33/core/index.html

```diff
@@ -4362,123 +4320,118 @@ <h4>Scripting Contexts</h4>
 							a future update adds the concept.</p>
 					</div>
 
-					<p>Which context a script is used in determines the rights and restrictions that a Reading System
-						places on it. Refer to <a href="#sec-scripted-container-constrained">the following sections</a>
-						and <a href="https://www.w3.org/TR/epub-rs-33/#sec-scripted-content-rs-reqs">Scripting
-							Conformance</a> [[EPUB-RS-33]] for some specific requirements that have to be adhered to
-						(not all Reading Systems will provide the same scripting functionality).</p>
+					<p>Authors should note that Reading Systems are expected to behave as though a unique <a
+							href="https://url.spec.whatwg.org/#origin">origin</a> [[URL]] has been assigned to each EPUB
```

Comment:
I know this is in line with the change on the origin, but I would prefer to see a stronger term than "expected", ie, "required".